src/App.tsx

```diff
@@ -15,48 +16,112 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+export enum Classes {
+  rev = 'reverse',
```

Comment:
You don't need the `rev` here. Add an additional state `isReversed` inside the App component